spec/test/stored_queries/test_fulltext_search_flags.py

```diff
@@ -0,0 +1,287 @@
+"""
+Tests the boolean flags on NCBI Taxon nodes when used in conjunction with full text search, e.g.
+in the queries:
+- taxonomy_search_species_strain
+- taxonomy_search_species_strain_no_sort
+
+Note these tests omit several fields from the records that are not relevant to the queries.
+"""
+
+import os
+import yaml
+from arango import ArangoClient
+from frozendict import frozendict  # type: ignore[attr-defined]
+from pathlib import Path
+from pytest import fixture
+from relation_engine_server.utils.config import get_config
+from relation_engine_client import REClient
+
+_QUERY_SORT = "taxonomy_search_species_strain"
+_QUERY_NO_SORT = "taxonomy_search_species_strain_no_sort"
+
+
+@fixture(scope="module")
+def conf():
+    config = get_config()
+    config["re_api_url"] = os.environ.get("RE_API_URL", "http://localhost:5000")
+
+    yield config
+
+
+@fixture(scope="module")
+def spec(conf):
+    # There should probably be some sort of abstraction here so this isn't hard coded, but AFAICT
+    # it doesn't exist in the code base at this point. Add later maybe. At least the tests
+    # should fail if the file is renamed / altered
+    schemapath = Path(conf["spec_paths"]["collections"]) / "ncbi" / "ncbi_taxon.yaml"
+    with open(schemapath) as fd:
+        spec = yaml.safe_load(fd)
+    yield spec
+
+
+@fixture(scope="module")
+def db(conf, spec):
+    # This is set to the "_system" db by the get_config() method defaults, which are used to start
+    # up the RE API in the docker tests...
+    arango = ArangoClient(conf["db_url"])
+    db = arango.db(conf["db_name"])
+
+    yield db
+
+    # clean up for tests in other modules that might be using the same db / collection
+    _clear_collection(db, spec["name"])
+
+
+@fixture(scope="module")
+def re_client(conf):
+    re_c = REClient(conf["re_api_url"], "non_admin_token")
+    yield re_c
+
+
+@fixture()
+def col(db, spec):
```

Comment:
I like `coll` so as not to confuse with `col` as in `col` and `row`

Reply:
Done